_posts/2021-02-16-release-1.12.2.md

```diff
@@ -0,0 +1,205 @@
+</li>
+</ul>
+
+<h2>        New Feature
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-20359'>FLINK-20359</a>] -         Support adding Owner Reference to Job Manager in native kubernetes setup
+</li>
+</ul>
+    
+<h2>        Improvement
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-9844'>FLINK-9844</a>] -         PackagedProgram does not close URLClassLoader
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-20417'>FLINK-20417</a>] -         Handle &quot;Too old resource version&quot; exception in Kubernetes watch more gracefully
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-20491'>FLINK-20491</a>] -         Support Broadcast Operation in BATCH execution mode
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-20517'>FLINK-20517</a>] -         Support mixed keyed/non-keyed operations in BATCH execution mode
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-20770'>FLINK-20770</a>] -         Incorrect description for config option kubernetes.rest-service.exposed.type
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-20907'>FLINK-20907</a>] -         Table API documentation promotes deprecated syntax
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21020'>FLINK-21020</a>] -         Bump Jackson to 20.10.5[.1] / 2.12.1
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21034'>FLINK-21034</a>] -         Rework jemalloc switch to use an environment variable
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21035'>FLINK-21035</a>] -         Deduplicate copy_plugins_if_required calls
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21036'>FLINK-21036</a>] -         Consider removing automatic configuration fo number of slots from docker
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21037'>FLINK-21037</a>] -         Deduplicate configuration logic in docker entrypoint
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21042'>FLINK-21042</a>] -         Fix code example in &quot;Aggregate Functions&quot; section in Table UDF page
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21048'>FLINK-21048</a>] -         Refactor documentation related to switch memory allocator 
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21123'>FLINK-21123</a>] -         Upgrade Beanutils 1.9.x to 1.9.4
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21164'>FLINK-21164</a>] -         Jar handlers don&#39;t cleanup temporarily extracted jars
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21210'>FLINK-21210</a>] -         ApplicationClusterEntryPoints should explicitly close PackagedProgram
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-21381'>FLINK-21381</a>] -         Kubernetes HA documentation does not state required service account and role
```

Comment:
This one (FLINK-21381) is committed to release-1.12 branch after RC1 is produced, and the current fix version of it is also 1.12.3, we should remove this one if the vote of RC1 passed.

Reply:
The issue is now included into RC#2.